### DISCUSSION TITLE
Alpine Collapse Plugin setStyles is not a function

### DISCUSSION THREAD

**Discussion by rizkhal** [alpinejs/alpine#2153] [1. Help] [answered]
Hello, i trying to use Collapse plugin in my project using npm [here](https://alpinejs.dev/plugins/collapse#installation)

And i copy and paste this example to test
```
<div x-data="{ expanded: false }">
    <button @click="expanded = ! expanded">Toggle Content</button>
 
    <p x-show="expanded" x-collapse>
        ...
    </p>
</div>
```

But when i click the button, i have error 
```
Alpine.setStyles is not a function
Expression: "expanded"
```

This is my dependencies
```
"@alpinejs/collapse": "^3.4.2",
"@tailwindcss/forms": "^0.3.3",
"@tailwindcss/typography": "^0.3",
"alpinejs": "^3.4.0",
"livewire-sortable": "^0.2.2",
"tailwindcss": "^2.1.4",
"tailwindcss-animatecss": "0.2.7",
"tailwindcss-multi-theme": "1.0.3"
```

And this my `resources/js/bootstrap.js` file
```
  window._ = require("lodash");
  
  import Alpine from "alpinejs";
  import collapse from "@alpinejs/collapse";
  import "livewire-sortable";
  
  Alpine.plugin(collapse);
  
  window.Alpine = Alpine;
  Alpine.start();
```

Whats wrong with it?

I already to clear the cache and reinstall dependency.

Thanks

**Comment by SimoTod** [comment, marked as answer]
You need to update Alpine (the core library) to the latest version (>= 3.4.2)

**Reply by rizkhal** [reply]
I still have the same error sir

![Screenshot from 2021-09-30 19-32-51](https://user-images.githubusercontent.com/24653114/135447898-d4625243-e470-4ede-823e-5f9263785aa3.png)

**Reply by rizkhal** [reply]
It's working now after i clear browser cache, thanks a lot dude!
